apps/measures/templates/apps_measures/measures_detail_page.html

```diff
@@ -0,0 +1,97 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+<div id="layout-grid__area--maincontent">
+    <h1>{{ page.page_title }}</h1>
+    <dl class="meta-information__row">
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Responsible organisations' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.responsible_orgs }}
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <div class="meta-information__column__head">
+                {% trans 'District' %}
+            </div>
+            <div class="meta-information__column__content">
+                {% with districts=page.get_districts_display %}
+                    {{ districts }}
+                {% endwith %}
+            </div>
+        </div>
+    </dl>
+    <dl class="meta-information__row">
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Involved organisations' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.involved_orgs }}
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Period' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.date }}
+            </dd>
+        </div>
+    </dl>
+    <dl class="meta-information__row">
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Handlungsfelder' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                tbd
+            </dd>
+        </div>
+        <div class="meta-information__column">
+            <dt class="meta-information__column__head">
+                {% trans 'Status' %}
+            </dt>
+            <dd class="meta-information__column__content">
+                {{ page.get_status_display }}
+            </dd>
+        </div>
+    </dl>
+    {% image page.text_image original as text_image %}
+    <div class="meta-information__image">
+        <img
+            src="{{ text_image.url }}"
+            alt="{{ text_image.alt }}"
+        />
+        {% if text_image.caption %}
+            <p class="image__caption">{{ text_image.caption }}</p>
+        {% endif %}
+    </div>
+    {{ page.body|richtext }}
+    {% if page.body_participation %}
```

Comment:
each of these richtext elements can go into a `<section>` element, I think they have an example of what classes they use in the style guide